evm-subql/project.yaml

```diff
@@ -3,7 +3,7 @@ description: ''
 repository: ''
 schema: ./schema.graphql
 network:
-  endpoint: ws://localhost:9944
+  endpoint: ws://host.docker.internal:9944
```

Comment:
should revert this. I think it is possible to override endpoint for subquery via cli args

Reply:
This persisted from debugging the macOS issues, I reverted it and verified that mac still has the expected behaviour.